app/javascript/css/tailwind.scss

```diff
@@ -213,15 +213,15 @@ a {
 }
 
 .button {
-    @apply bg-grey-dark text-white ml-4 py-2 px-3 no-underline ;
+    @apply bg-grey-darker text-white font-bold py-2 px-5 no-underline ;
 
     &:hover {
         @apply bg-grey-darker;
     }
 }
 
 .shared_devise_link {
-    @apply bg-grey-dark text-white ml-4 py-2 px-3 no-underline ;
+    @apply bg-grey-darker text-white font-bold py-2 px-5 no-underline ;
```

Comment:
check this is not used in other views that require the original styling

Reply:
This is only currently being used with the log in and sign up buttons and doesnt effect anything else